.circleci/workflows.yml

```diff
@@ -1782,6 +1782,8 @@ jobs:
             - run: yarn lerna run types
             - sanitize-verify-and-store-mocha-results:
                 expectedResultCount: 20
+            - store_artifacts:
+                path: /tmp/cypress/junit
```

Comment:
There's no way to debug which unit-tests didn't run without this.

Reply:
Ahhh, I couldn't remember why we removed it! Now I remember. I'll remove it.

Reply:
removed here: [`c0bfbde` (#31518)](https://github.com/cypress-io/cypress/pull/31518/commits/c0bfbde2beae7fcd079d72b6d26eee0da7b2667f)